src/orquestra/sdk/exceptions.py

```diff
@@ -83,6 +83,10 @@ class InvalidTaskDefinitionError(BaseRuntimeError):
     pass
 
 
+class NodesInTaskResourcesWarning(Warning):
```

Comment:
**suggestion**: could you add a docstring? The name suggests that it's about having nodes in task's resources, but it might be worthwhile describing why it's a bad thing.